keeper-gradle-plugin/src/main/kotlin/com/slack/keeper/VariantClasspathJar.kt

```diff
@@ -66,7 +66,7 @@ abstract class VariantClasspathJar : DefaultTask() {
 
   fun from(vararg paths: Any) {
     IntRange(0, 3).forEach {
```

Comment:
nit: let's explain why this is so magic in a comment

```suggestion
    // This is magic
    IntRange(0, 3).forEach {
```